Source/Example Measurements/NI-FGEN Standard Function/README.md

```diff
@@ -0,0 +1,27 @@
+## NI-FGEN Standard Function
+
+This is a MeasurementLink example that generates a waveform using standard function mode with an NI FGEN.
+
+### Features
+
+- Uses the NI-FGEN LabVIEW API
+- Pin-aware, supporting one session and one pin
+  - Generates the same waveform on all selected pin/site combinations
+- Includes InstrumentStudio project file
+- Includes a TestStand sequence showing how to configure the pin map, register
+  instrument sessions with the session management service, and run a measurement.
+  - For the sake of simplicity, the TestStand sequence handles pin map and session registration and unregistration in the `Setup` and `Cleanup` sections of the main sequence. For **Test UUTs** and batch process model use cases, these steps should be moved to the `ProcessSetup` and `ProcessCleanup` callbacks.
+- Uses the NI gRPC Device Server to allow sharing instrument sessions with other
+  measurement services when running measurements from TestStand
+
+### Required Driver Software
+
+- NI-FGEN 2023 Q1 or later
+
+### Required Hardware
+
+This example requires an NI FGEN that is supported by NI-FGEN (e.g. PXIe-5433).
+
+To simulate an NI FGEN in software: open `NI Hardware Configuration Utility`, click `Edit` -> `Add Hardware...`,
```

Comment:
Does the Hardware Configuration Utility come with the latest version of NI-FGEN?

Reply:
I'm not sure, I can look into that. The DCPower readme suggests using MAX but that isn't applicable to NI-FGEN since no devices support simulation via MAX for FGEN.

Reply:
I wasn't able to find anything that definitively said config utility gets installed with the drivers and getting a system set up just to test that hypothesis didn't seem worth the time investment. Since MAX isn't a useful alternative for simulating FGEN devices I'll leave this as is.